src/environments/environment.prod.ts

```diff
@@ -2,6 +2,7 @@ import { Profile } from '../app/auth/profiles/profiles.component';
 
 export const AppConfig = {
   production: true,
+  test: false,
```

Comment:
Sorry - I introduced this `test` property in the production AppConfig.
However, we could actually omit `test` here, as well as in the default environment.
I think this will be simpler, what do you think?

Reply:
Yes that sounds good!

Reply:
Should `production` be removed from the default environment as well?

Reply:
Removing it from the default AppConfig causes our Mock Service Factories to fail since they check for the existence of the `test` property even when serving the default config.

Reply:
Yes, my bad - it'll lead to a type error. Let's keep `test` properties in all configs, to be consistent then?

Reply:
Sure 👍